pyfiat/client.py

```diff
@@ -161,14 +166,20 @@ def _update_vehicle(v: Vehicle, p: dict) -> Vehicle:
         vi, "distanceToService", "distanceToService", "value")
     v.distance_to_service_unit = sg(
         vi, "distanceToService", "distanceToService", "unit")
-    v.distance_to_empty = sg(batt, "distanceToEmpty", "value")
-    v.distance_to_empty_unit = sg(batt, "distanceToEmpty", "unit")
-
+    v.distance_to_empty_gas = sg(vi, "fuel", "distanceToEmpty", "value")
+    v.distance_to_empty_gas_unit = sg(vi, "fuel", "distanceToEmpty", "unit")
+    v.distance_to_empty_ev = sg(batt, "distanceToEmpty", "value")
+    v.distance_to_empty_ev_unit = sg(batt, "distanceToEmpty", "unit")
+    v.fuel_low_gas = sg(vi, "fuel", "isFuelLevelLow")
```

Comment:
Let's remove the `_gas` prefix here maybe since it's anyway obvious that fuel relates to a gas-vehicle :)